docs/topics/proc_running-the-secured-booster-integration-tests-on-vx.adoc

```diff
@@ -0,0 +1,30 @@
+[#running-the-secured-booster-integration-tests_context]
```

Comment:
Ah! Should `context` here be actually `{context}`?

Reply:
I don't know (nothing really changed when I built it with `{context}`). If so, then also the `proc_running-the-secured-booster-integration-tests-on-sb.adoc` needs to be updated, as it wasn't there before.

Reply:
It was probably wrong, but I'm not 100% sure. We'll need someone from the docs team to confirm.

Reply:
@spisiakm I think @Ladicek has a good point, `context` here should be `{context}`, that is, it should be marked up as a variable, as this is a measure that ensures module  reusability while avoiding section ID conflicts. The reason it does not get rendered every time is that we define it as an empty variable if  the module is not being used multiple times. If it is not defined in the `master.adoc` file it will simply not show up. but it's still there:) Does that make sense?

Reply:
@inoxx03 Thing is, the files are now runtime-specific and there's no reuse anymore. Do you still use `{context}` in such situations? Thanks!

Reply:
If the module is runtime specific and only used i one place, then I'd be inclined to omit `{context}` and use the designator for the appropriate runtime. in this case, the ID would be: `[#running-the-secured-booster-integration-tests_vertx]`

Reply:
@rhoads-zach @Ladicek If I change `context` to `{context}`, the script `validate_guides.sh` fails to validate vertx-runtime:
```shell
master.xml:3499: element section: Schemas validity error : 
Element '{http://docbook.org/ns/docbook}section', 
attribute '{http://www.w3.org/XML/1998/namespace}id': 'running-the-secured-booster-integration-tests_{context}' is not a valid value of the atomic type 'xs:ID'.
```
The same applies to spring-boot-runtime (wrapping `context` in braces).

Reply:
I think, given that those files are no longer shared, we can just hardcode the value. So instead of `context`, use `sb` or `vertx` or something like that.

Reply:
+1, I would use just `sb` or `vertx` as well. I'll push an update if nobody is against it.

Reply:
+1 to using hardcoded values if the module is not reused. @spisiakm FYI The reason why the validation fails when `{context}` is wrapped in curly braces is likely because the ID can not contain a variable that is not defined. It can however contain an empty variable. So, as a workaround, you can set `:context:` to be empty in the master file. this should fix the issue. That being said, please use hardcoded values i this specific instance :)